backend/model/seriesSchema.js

```diff
@@ -0,0 +1,129 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const seriesSchema = new Schema(
+  {
+    seriesTitle: {
+      type: String,
+      minLength: [5, "Series title must be at least 5 characters"],
+      maxLength: [40, "Series title must be less than 40 characters"],
+      required: [true, "Series title is must be provide"],
+    },
+    seriesSummary: {
+      type: String,
+      minLength: [15, "Series summary must be at least 15 characters"],
+      maxLength: [250, "Series summary must be less than 250 characters "],
+    },
+    genres: {
+      type: [String],
+      enum: [
+        "Action",
+        "Anime",
+        "Children & Family",
+        "Classic",
+        "Comedies",
+        "Documentaries",
+        "Dramas",
+        "Horror",
+        "Romantic",
+        "Sci-fi & Fantasy",
+        "Sports",
+        "Thrillers",
+        "Adventure",
+      ],
+    },
+    releaseDate: {
+      type: Date,
+      required: [true, "Content release date is required!"],
+    },
+    cast: [String],
+    director: {
+      type: String,
+      required: [true, "Director is required"],
+    },
+    rating: {
+      type: String,
+      required: true,
+    },
+    language: {
+      type: String,
+      enum: [
+        "English",
+        "Hindi",
+        "Korean",
+        "Japan",
+        "Tamil",
+        "Spanish",
+        "German",
+      ],
+    },
+    thumbnail: [
```

Comment:
thumbnail URL and thumbnail id must be required I believe

Reply:
seriesSchema is removed.